flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/TaskIOMetricGroup.java

```diff
@@ -100,6 +104,10 @@ public TaskIOMetricGroup(TaskMetricGroup parent) {
                         hardBackPressuredTimePerSecond::getMaxSingleMeasurement);
 
         this.busyTimePerSecond = gauge(MetricNames.TASK_BUSY_TIME, this::getBusyTimePerSecond);
+
+        this.mailboxThroughput = meter(MetricNames.MAILBOX_THROUGHPUT, new MeterView(60));
+        this.mailboxLatency =
+                histogram(MetricNames.MAILBOX_LATENCY, new DescriptiveStatisticsHistogram(60));
```

Comment:
Why are you defining `60s`  as `timeSpanInSeconds` here? Why not using a default parameter?

Reply:
In fact, it is the default parameter. The odd thing is that the constructor of MeterView requires either new SimpleCounter() or timeSpanInSeconds as parameter. So it would be necessary to add std constructors to both MeterView and DescrHistogram. However, I thought this is the best place to set the value and have it consistent for both mailbox metrics even though it's hard coded.